Sources/XcodeGenKit/PBXProjGenerator.swift

```diff
@@ -568,6 +568,7 @@ public class PBXProjGenerator {
             (try path.children().sorted(), path)
 
         let excludedFiles: [String] = [".DS_Store"]
```

Comment:
Just a note, these default excludedFiles should be moved up into the spec options at some point.